### PR TITLE
Fixup campaign handling

### PR DESCRIPTION
**Pull request by ocehugo**
last round-up of fixes regarding campaigns. See commit msg.

### CODE REVIEW

**Comment by ocehugo** [comment]
@lbesnard, @mhidas 

This is ready to go and fully tested with pipelinebox+stack.

**Comment by mhidas** [comment]
:+1: Looks ok to me, but I'll let @lbesnard review and merge it, as hopefully he's more familiar with the logic of what this is supposed to do.

File: test_aodndata/aatams/test_aatams_sattag.py [hunk 1]
```diff
@@ -27,6 +27,8 @@
 NRT_EMPTY_DIVE_ZIP = os.path.join(TEST_ROOT, "emptydive_ct999_nrt.zip")
 NRT_MIXED_CAMPAIGN = os.path.join(TEST_ROOT, "mixcampaign_ct555_nrt.zip")
 NRT_CSV_CAMPAIGN_MISMATCH = os.path.join(TEST_ROOT, "csvcampaignmismatch_ct999_nrt.zip")
+NRT_CSV_CAMPAIGN_FILE_CONTENT_MISMATCH = os.path.join(TEST_ROOT, "csvfilecontentmismatch_ct156_nrt.zip")
+
```
Comment (lbesnard) [inline test_aodndata/aatams/test_aatams_sattag.py:31]: delete this line

File: test_aodndata/aatams/test_aatams_sattag.py [hunk 2]
```diff
@@ -164,13 +166,17 @@ def test_stream_update(self):
...
-        """Test block of a bad zip with multiple campaigns csv filenames"""
+        """Test a bad zip with multiple campaigns csv filenames"""
         self.run_handler_with_exception(SchemaError, NRT_MIXED_CAMPAIGN)
 
     def test_csvcampaignmismatch(self):
-        """Test block of a bad csv file with mismatch campaign filename and csv campaign content"""
+        """Test a bad zip file with consistent campaign filenames but inconsistent csv campaign content"""
         self.run_handler_with_exception(SchemaError, NRT_CSV_CAMPAIGN_MISMATCH)
 
+    def test_csvcampaignmismatch(self):
```
Comment (lbesnard) [inline test_aodndata/aatams/test_aatams_sattag.py:176]: same function name as above

**Review by lbesnard — CHANGES_REQUESTED** [review]

File: aodndata/aatams/aatams_sattag_schema.py [hunk 3]
```diff
@@ -681,6 +687,7 @@ def __init__(self, skip_every=0, bulk_load=True, dialect_csv=None):
         self.summary = {}
 
 
+
```
Comment (lbesnard) [inline aodndata/aatams/aatams_sattag_schema.py:690]: same,no need for 3 empty lines

File: test_aodndata/aatams/test_aatams_sattag.py (hunk 1 above)
Comment (ocehugo) [reply test_aodndata/aatams/test_aatams_sattag.py:31]: done

File: test_aodndata/aatams/test_aatams_sattag.py (hunk 2 above)
Comment (ocehugo) [reply test_aodndata/aatams/test_aatams_sattag.py:176]: done

File: aodndata/aatams/aatams_sattag_schema.py (hunk 3 above)
Comment (ocehugo) [reply aodndata/aatams/aatams_sattag_schema.py]: done

**Comment by lbesnard** [comment]
@ocehugo  the build failed but you merged it. See:
http://build.aodn.org.au/job/python-aodndata_build/job/fixup_campaign_handling/5/console

**Pull request merged**